### PR TITLE
refactor(frontend): Blockchain API URL is hard-coded

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

The Blockchain API URL can be hard-coded since it is constant throughout the envs.

### CODE REVIEW

File: src/frontend/src/env/rest/blockchain.env.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { UrlSchema } from '$lib/validation/url.validation';
+
+export const BLOCKCHAIN_API_URL = UrlSchema.parse('https://blockchain.info');
```
Comment (peterpeterparker) [inline src/frontend/src/env/rest/blockchain.env.ts:3]: same as https://github.com/dfinity/oisy-wallet/pull/3264#discussion_r1825452517

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/env/rest/blockchain.env.ts]: after that PR, i changed here too

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Review by DenysKarmazynDFINITY — APPROVED** [review]
Thanks!

**Pull request merged**